hbase/src/main/resources/hbase/bin/hirb.rb

```diff
@@ -0,0 +1,189 @@
+#
```

Comment:
I think we should avoid committing a copy here to reduce maintenance cost. Could/should this be in the interpreter doc (it would be great if you could take a shot at creating one) on how to get the one from hbase distribution instead?